src/test/scala/uk/gov/hmrc/test/ui/pages/section5/ProcedureCodesPage.scala

```diff
@@ -69,5 +68,7 @@ object ProcedureCodesPage extends BasePage {
 
   def hasRestrictingZeroVatPC: Boolean = codesRestrictingZeroVat.contains(detail(ProcedureCode(itemId)))
 
-  def isPermanentExportOfUKGoodsPC: Boolean        = detail(ProcedureCode(itemId)) == "1040"
-}
\ No newline at end of file
+  def isPermanentExportOfUKGoodsPC: Boolean = listOfItemDetailFor(ProcedureCodeLabel).contains("1040")
+
+  def isSupervisingCustomsOfficePageVisiblePC: Boolean = detail(ProcedureCode(itemId)) == "1042"
```

Comment:
WDYT of using `has` instead of `is` for these methods?

`isPermanentExportOfUKGoodsPC` => `hasPermanentExportOfUKGoodsPC`
`isSupervisingCustomsOfficePageVisiblePC` => `hasSupervisingCustomsOfficePageVisiblePC`

Reply:
Done